src/platform_impl/ios/window.rs

```diff
@@ -452,7 +452,7 @@ impl Window {
             // Like the Windows and macOS backends, we send a `ScaleFactorChanged` and `Resized`
             // event on window creation if the DPI factor != 1.0
             let scale_factor: CGFloat = msg_send![view, contentScaleFactor];
-            let scale_factor: f64 = scale_factor.into();
+            let scale_factor: f64 = scale_factor as _;
```

Comment:
The variable type was removed in a similar expression higher up.

Reply:
I don't understand what do you want from me here. The problem is that CGFLoat is either f32 or f64, but winit API is f64, so we need to cast it to f64 here. Using `.into()` won't work, since clippy finds it as redundant when `CGFloat` is `f64`, however we can cast it like `as _` and explicitly state that we assign to `f64` value.